app/views/contexts/calculated_summary_context.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         metadata: MetadataProxy | None,
         response_metadata: MutableMapping,
         routing_path: RoutingPath,
-        current_location: LocationType,
+        current_location: Location,
```

Comment:
Should this have changed?. Couple of others like this, bad merge?. These were changed in the type hinting PR.

Reply:
Yeah this is from that big merge, fixed now, thanks!